lib/page/capture_page.dart

```diff
@@ -99,133 +107,131 @@ class _CapturePageState extends State<CapturePage> {
 
   @override
   Widget build(BuildContext context) {
+    if (widget.cameras?.isEmpty ?? true) {
+      return Scaffold(
+        appBar: AppBar(),
+        body: const Center(
+          child: Text('No camera found for this device'),

```

Comment:
Please use the i10n mechanism:

1. First add a new key (newAddedKey) to the `lib/l10n/app_en.arb` and `lib/l10n/app_fr.arb` files.
2. Then use it in dart file : `AppLocalizations.of(context)!.newAddedKey`

Reply:
Oh yes, nice catch! 
I forgot this, sorry.